cloud.md

```diff
@@ -1,7 +1,15 @@
 # Cloud
 
+Once the virtual machines are instantiated in the cloud give some time for their kernel get updated. To re-use the same inventory file update the **/etc/hosts** with the respective IPs for each host.
```

Comment:
On SURFSara HPC cloud the machine get an DNS entry as `${vmname}.${projectname}.surf-hosted.nl`.
See https://doc.hpccloud.surfsara.nl/access-your-VM. So the **/etc/hosts** does not need to be edited.